packages/react-core/src/components/DataList/__tests__/Generated/DataList.test.tsx

```diff
@@ -15,6 +15,7 @@ it('DataList should match snapshot (auto-generated)', () => {
       aria-label={'string'}
       onSelectDataListItem={(id: string) => undefined as void}
       selectedDataListItemId={"''"}
+      isCompact={false}
```

Comment:
Looks like this is updating tests outside of Table as well, would be good if this PR only focused on the Table, perhaps spin these other ones out into a separate PR?